rust/Dockerfile

```
@@ -2,7 +2,7 @@
 ARG FROM
 FROM $FROM
 
-RUN echo "bumping on 2022/01/22"
+RUN echo "bumping on 2022/10/29"
```

```suggestion
RUN echo "bumping on 2023/03/08"
```

My fault ig guess for leaving overlooking this. While at it, it should be the most recent one to date ig.